src/reference/02-DetailTopics/05-Plugins-and-Best-Practices/02-Plugins.md

```diff
@@ -153,10 +142,19 @@ integrate.
 To make an auto plugin, create a project and enable `SbtPlugin`.
 
 ```scala
+ThisBuild / version := "0.1.0-SNAPSHOT"
+ThisBuild / organization := "com.example"
+ThisBuild / homepage := Some(url("https://github.com/sbt/sbt-hello"))
+
 lazy val root = (project in file("."))
   .enablePlugins(SbtPlugin)
   .settings(
-    name := "sbt-something"
+    name := "sbt-hello",
+    pluginCrossBuild / sbtVersion := {
+      scalaBinaryVersion.value match {
+        case "2.12" => "1.2.8" // set minimum sbt version
```

Comment:
Can we add an explanation about this part in the text immediately before or after this snippet? What does it do?

The line comment explains a bit, however it left me wondering: shouldn't one be setting a *maximum* sbt version, given the used scala version?